alpha-build-machinery/make/targets/golang/build.mk

```diff
@@ -1,7 +1,8 @@
 self_dir :=$(dir $(lastword $(MAKEFILE_LIST)))
 
+go_bin_suffix ?=
 define build-package
-	$(strip $(GO) build $(GO_BUILD_FLAGS) $(GO_LD_FLAGS) $(1))
+	$(strip $(GO) build $(GO_BUILD_FLAGS) $(GO_LD_FLAGS) $(if $(go_bin_suffix),-o '$(notdir $(1))$(go_bin_suffix)',) $(1))
```

Comment:
what does it do? worth a comment?

Reply:
fills `-o` only if `$(go_bin_suffix)` is not empty